DependencyInjection/Configuration.php

```diff
@@ -107,7 +107,14 @@ public function getConfigTreeBuilder()
                                 return $v;
                             })
                         ->end()
-                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))->end()
+                        ->defaultValue(class_exists('Doctrine\ORM\EntityManager'))
+                    ->end()
+                    ->arrayNode('service_naming_strategy')
+                        ->children()
+                            ->scalarNode('id')->end()
+                            ->scalarNode('class')->end()
```

Comment:
I would remove the `id` and `class` children and just change it to something like this:
`->scalarNode('service_naming_strategy')->devaultValue('jms_di_extra.service_naming_strategy.default')`